packages/common/src/api/tan-query/utils/primeTrackData.ts

```diff
@@ -20,7 +20,7 @@ export const primeTrackData = ({
   forceReplace = false,
   skipQueryData = false
 }: {
-  tracks: (UserTrackMetadata | TrackMetadata)[]
+  tracks: (UserTrackMetadata | Track)[]
```

Comment:
sup with this?

Reply:
the `_marked_deleted` property only exists on Track (via ComputedTrackProperties) though this might be polluting the query store a bit 🤔